tensorflow/compiler/xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -301,6 +301,44 @@ llvm::Type* GetIndexTypeForKernel(const HloInstruction* hlo, int64 launch_size,
   return b->getInt32Ty();
 }
 
+// Get the input shape of the ROOT slices, which will be used as the kernel
```

Comment:
Another nitpick: for function docstrings we use declarative rather than imperative, cf. https://google.github.io/styleguide/cppguide.html#Function_Comments

This is applicable here and below.

Reply:
Got it. Will do.

Reply:
Done.